### PR TITLE
Chinese: Istio 0.7 realease note

### PR DESCRIPTION
**Pull request by gxian**
Translate 0.7 release note into Chinese

### CODE REVIEW

**Comment by gxian** [comment]
I signed it!

**Comment by rootsongjc** [comment]
@gxian Please sync with master and push again to trigger the CI.

**Comment by gxian** [comment]
- ./public/about/contribute/writing-a-new-topic/index.html
  *  External link https://gohugo.io/content-management/front-matter/ failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: SSL connect error

**Comment by gxian** [comment]
I pushed again and CI tests failed, any suggestion?

**Comment by rootsongjc** [comment]
@gxian It's a problem of CI, please change something and submit a commit to trigger the CI again.

File: content_zh/about/notes/0.7/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+尽管如此，这次发布包含了大量的问题修复和性能提升。
+
+请注意该发布包含新的v1alpha3流量管理功能的初步支持，这个功能尚处于频繁变动当中并且可能会在0.8版本存在不兼容的变更。
+所以如果您希望尝试，敬请自便，但是请预期这些功能可能会在0.8或者之后的版本发生变动。
+
+已知问题:
+
+我们的 [helm chart](/docs/setup/kubernetes/helm-install/)
+目前需要一些变通的方式才能正确工作，这里 [4701](https://github.com/istio/istio/issues/4701) 有相关细节。
```
Comment (rootsongjc) [inline content_zh/about/notes/0.7/index.md]: CI 已经通过了，但是需要修改下内容：
这里 [4701](https://github.com/istio/istio/issues/4701) -> [Issue 4701](https://github.com/istio/istio/issues/4701)

Comment (gxian) [reply content_zh/about/notes/0.7/index.md]: 修改完毕，已Push

**Pull request merged**